src/i18n/strings/es.ts

```diff
@@ -164,4 +164,15 @@ export const templates = {
   sfc7214f623fe475d: `Selecciona la empresa a la que paga su factura de electricidad.`,
   sfe16afc784bb9d76: `Tejado solar`,
   sfe81d5d73a35a2cf: `Calcular`,
+  sb1ef6ac20f1ddfff: `Discount off an electric panel`,
```

Comment:
Are these just placeholders as we wait for the translated strings? How do we usually manage translations?

Reply:
I was waiting until we got agreement on copy to request translations. This is just what happens if you run `yarn strings:build` without translations: the original string gets put in as a fallback. I'll wait until we get translations to land this.